### PR TITLE
Vote-1130: Life Events Refinements

### PR DESCRIPTION
**Pull request by clmedders**
## Jira ticket

[Vote-1130](https://cm-jira.usa.gov/browse/VOTE-1130)

## Description

Style Refinements for life events section on homepage.

Changes applied
- updated content to match design 
- updated link to added hover effect 
- adjusted column use and offset 
- adjusted padding for break points 

[Figma Link](https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=8012-20081&m=dev)

### Post-deploy steps

1. run `lando retune` then cd into `votegov` theme and run `npm run build`

### QA/Testing instructions

1. log into site and update the quick links section content to match design -> Heading: Have you changed your name, address, or party affiliation?, Content: You'll select your state or territory to get started., Link: Url `/register` Link Text - Check or update your registration
2. [figma Link](https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=8012-20081&m=dev) for reference - review the default section on the far left at the various break points -> Mobile, tablet and desktop 
3. check that hover effect is working as expected (adding underline and arrow moves from 4px to 12px)
4. check that no major visual regressions appear 

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [ ] The file changes are relevant to the task objective.
- [ ] Code is readable and includes appropriate commenting.
- [ ] Code standards and best practices are followed.
- [ ] QA/Test steps were successfully completed, if applicable.
- [ ] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 1]
```diff
@@ -11,28 +11,59 @@
...
 
   @include at-media('desktop') {
-    @include grid-col(5);
+    @include grid-col(6);
   }
 }
 
 .vote-quicklinks__content{
   @include u-font('sans', 'md');
+  @include u-text('normal');
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/components/quick-links.scss:43]: You can remove this, the inherited style is the same.

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 2]
```diff
@@ -11,28 +11,59 @@
...
 
   a {
-    color: $ac-cool-light !important;
-  }
+    @include u-text('bold');
+    color: $base-white !important;
+    text-decoration: none;
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/components/quick-links.scss:48]: Use the USWDS mixin

**Review by SamiraMSadat — CHANGES_REQUESTED** [review]

File: web/themes/custom/votegov/src/sass/components/quick-links.scss (hunk 1 above)
Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/quick-links.scss:43]: line removed

File: web/themes/custom/votegov/src/sass/components/quick-links.scss (hunk 2 above)
Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/quick-links.scss]: thanks for finding that mixin!

**Review by SamiraMSadat — DISMISSED** [review]

**Review by rayestrada — APPROVED** [review]

**Pull request merged**